src/test/java/Homework19.java

```diff
@@ -0,0 +1,26 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework19 extends BaseTest {
+
+    @Test
+    public void newSongToPlaylist() throws InterruptedException {
+        //etUpBrowser();
+        //String url = "https://bbb.testpro.io/";
+        //driver.get(url);
+        //getBrowser();
+        emailField("skillztomkid@gmail.com");
+        passwordFinder("te$t$tudent");
+        logIn();
+        searchSongs("dark days");
+        viewAll();
+        clickFirstSong();
+        addToPlaylist();
+        newPlaylist("Ope");
+        opePlaylist();
```

Comment:
Test failed! You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/223355670-18aef97f-2d0f-40a5-a702-f8b9b0849d69.png)